internal/pkg/local/manager.go

```diff
@@ -200,6 +205,18 @@ func (u *UnitOfWork) DeleteObject(objectState model.ObjectState, record model.Re
 		})
 }
 
+func (u *UnitOfWork) Rename(actions []model.RenameAction) {
+	u.
+		workersFor(1000). // rename at the end
+		AddWorker(func() error {
+			if err := u.rename(actions); err != nil {
+				return err
+			}
+			u.renamed = append(u.renamed, actions...)
```

Comment:
Ak sa `rename` podarilo, tak si to poznacime, a na koniec `Invoke` zavolame `mapper.OnObjectsRename`.